src/utils/Utils.js

```diff
@@ -2,7 +2,7 @@
  * @flow
  */
 
-export function randomId() :string {
+export default function randomId() :string {
```

Comment:
No need to add default here since this is a utilities file that is meant to export more than one utility function. If anything, I'd rather remove `randomId()`.